model/godleyIcon.h

```diff
@@ -78,6 +78,10 @@ namespace minsky
 
     /// sets editor's display values attributes to current global preferences
     void setEditorDisplayValues();
+    
+    CopiableUniquePtr godleyT;
+    
+    std::string classType() const override {return "GodleyIcon";}    
```

Comment:
Oops - you removed this in Feature 17, but this PR looks like it will reinstate it.